_cucumber/lib/pages/products_overview_page.rb

```diff
@@ -2,7 +2,7 @@
 
 class ProductOverviewPage < SiteBase
 
-  element(:side_nav)  { |b| b.ul(class: '.side-nav .rhd-sub-nav') }
+  element(:side_nav) { |b| b.ul(css: '.side-nav .rhd-sub-nav') }
```

Comment:
@adelasofia did you try this locally? I just tried manually and this css selector is not found hence the acceptance test fails.

I think you should use:
element(:side_nav) { |b| b.ul(class: 'rhd-sub-nav') }